lib/circuitry/subscriber.rb

```diff
@@ -12,15 +12,16 @@ class Subscriber
     include Concerns::Async
     include Services::SQS
 
-    attr_reader :queue, :timeout, :wait_time, :batch_size, :lock, :ignore_visibility_timeout
+    attr_reader :queue, :timeout, :wait_time, :batch_size, :lock, :ignore_visibility_timeout, :async_delete
 
     DEFAULT_OPTIONS = {
       lock: true,
       async: false,
       timeout: 15,
       wait_time: 10,
       batch_size: 10,
-      ignore_visibility_timeout: false
+      ignore_visibility_timeout: false,
+      async_delete: false
```

Comment:
lets call this something other than `async_delete` -- to me that setting would imply that the delete still happens but is somehow non-blocking for the processing of further messages.

Suggest: `auto_delete` with a default setting of true.

Reply:
👍